src/components/scale.jsx

```diff
@@ -0,0 +1,57 @@
+import classNames from "classnames"
+import React from "react"
+import {FormattedDate} from 'react-intl';
+
+import * as AppPropTypes from "../app-prop-types"
+import Value from "./value"
+
+
+const Scale = React.createClass({
+  propTypes: {
+    parameter: AppPropTypes.parameter.isRequired,
+  },
+  render() {
+    const {brackets} = this.props.parameter
+    return (
+      <table className="table table-bordered table-hover table-striped">
+        <tbody>
+          {
+            Object.keys(brackets).sort().reverse().map((startDate, startDateIndex) => {
+              const bracketAtDate = brackets[startDate]
+              return Object.keys(bracketAtDate).map((threshold, thresholdIndex) => {
+                const value = bracketAtDate[threshold]
+                return (
+                  <tr
+                    className={classNames({"first-bracket": thresholdIndex === 0})}
+                    key={`${startDateIndex}-${thresholdIndex}`}
+                  >
+                    {
+                      thresholdIndex === 0 && (
+                        <td rowSpan={Object.keys(bracketAtDate).length}>
+                          À partir du
+                          {" "}
+                          <FormattedDate value={startDate} />
```

Comment:
Yes because otherwise the space will not be in the DOM. That's a limitation of the React virtual DOM engine (see [this issue](https://github.com/facebook/react/issues/1643)).

The other way to avoid the problem is to use a `<span>` but this produces a heavier DOM.